apps/console/src/components/pages/auth/subscriber-verify/verifier.tsx

```diff
@@ -50,34 +170,43 @@ export const TokenVerifier = () => {
     return (
       <div className="flex flex-col m-auto self-center">
         <div className="mx-auto animate-pulse w-96">
-          <Logo />
+          <Logo width={213} />
         </div>
-        <div className={errorMessage()}>{error}</div>
+        <div className={messageWrapper()}>
+          <CircleX size={37} className="text-destructive" strokeWidth={1.5} />
+          <div>
+            <p>{error}</p>
+            <p>If you continue to have issues, please reach out to our support team.</p>
+          </div>
+        </div>
+
+        <Button className="size-fit mt-4 mx-auto mb-5">Send email to support team</Button>
```

Comment:
This button is missing the link, should be able to use `href={SUPPORT_EMAIL} ` ; Lets also update the text
```suggestion
        <Button className="size-fit mt-4 mx-auto mb-5">Contact Support</Button>
```